engine/runtime/src/main/java/module-info.java

```diff
@@ -0,0 +1,7 @@
+module org.enso.runtime {
+  requires java.base;
+  requires org.graalvm.polyglot;
+  requires org.graalvm.truffle;
+  provides com.oracle.truffle.api.provider.TruffleLanguageProvider with
+      org.enso.interpreter.EnsoLanguageProvider;
```

Comment:
There was a discussion asking whether it is better to have a single `module-info.java` for the whole `runtime.jar` or whether we should create multiple JARs. I prefer to keep _status quo_ to minimize the number of changes. E.g. if we can let's keep single `runtime.jar` with a single `module-info.java`.

Pavel mentioned there is a problem with compilation. Yes, I can see it. Here is my sample `module-info.java`:
```java
module org.enso.runtime {
    requires org.graalvm.truffle;

    provides com.oracle.truffle.api.TruffleLanguage.Provider with
        org.enso.interpreter.epb.EpbLanguageProvider,
        org.enso.interpreter.EnsoLanguageProvider;

}
```
trying to compile it with `runtime.jar` on classpath doesn't work:
```bash
enso/t$ javac -d target module-info.java -cp ../runtime.jar
odule-info.java:5: error: package org.enso.interpreter.epb is not visible
        org.enso.interpreter.epb.EpbLanguageProvider,
                            ^
  (package org.enso.interpreter.epb is declared in the unnamed module, but module org.enso.runtime does not read it)
module-info.java:6: error: package org.enso.interpreter is not visible
        org.enso.interpreter.EnsoLanguageProvider;
                ^
  (package org.enso.interpreter is declared in the unnamed module, but module org.enso.runtime does not read it)
```
however there is a trick to do it compile successfully:
```bash
enso/t$ unzip ../runtime.jar -d target
enso/t$ javac -d target module-info.java
enso/t$ ls target/module-info.class 
target/module-info.class
```
Let's use this simple trick for now.